coding-intro/coding-intro-core/core-concepts/programs-and-data.md

```diff
@@ -5,6 +5,10 @@ type: normal
 
 category: must-know
 
+links:
+  - >-
+    [EXACT Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){website}
```

Comment:
let's make the tag `video`

```suggestion
    [EXACT Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){video}
```

we can also say `'Exact'`. it looks better imo but it's up to you if you want to keep it as `'EXACT'`

```suggestion
    [Exact Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){video}
```